examples/webgl_materials_envmaps_hdr.html

```diff
@@ -252,20 +255,28 @@
 
 					standardMaterial.roughness = params.roughness;
 					standardMaterial.bumpScale = - 0.05 * params.bumpScale;
+
 					var newEnvMap = standardMaterial.envMap;
+
 					switch( params.envMap ) {
+
 						case 'None': newEnvMap = null; break;
-						case 'LDR': newEnvMap = ldrCubeRenderTarget; break;
-						case 'HDR': newEnvMap = hdrCubeRenderTarget; break;
-						case 'RGBM16': newEnvMap = rgbmCubeRenderTarget; break;
+						case 'LDR': newEnvMap = ldrCubeRenderTarget ? ldrCubeRenderTarget.texture : undefined; break;
+						case 'HDR': newEnvMap = hdrCubeRenderTarget ? hdrCubeRenderTarget.texture : undefined; break;
+						case 'RGBM16': newEnvMap = rgbmCubeRenderTarget ? rgbmCubeRenderTarget.texture : undefined; break;
```

Comment:
When it's tidying / shows better practice, I guess it's fine. Where's the code?

Just forward-push it in a separate commit? Then you can easily take it out again via `git reset HEAD~1` (or `git rebase -i <base commit>` if it's not the last - I recently wrote a [Wiki page](https://github.com/mrdoob/three.js/wiki/A-Contributor's-Guide-to-the-Git-Galaxy) that explains changing the history in a little more detail).